autosklearn/pipeline/base.py

```diff
@@ -73,10 +73,6 @@ def __init__(self, config=None, steps=None, dataset_properties=None,
 
         self.set_hyperparameters(self.config, init_params=init_params)
 
-        if random_state is None:
-            self.random_state = check_random_state(1)
-        else:
-            self.random_state = check_random_state(random_state)
```

Comment:
This is now done somewhere else I guess?

Reply:
Following the comments previously, we don't want to turn it into a `RandomState` object ourselves. The BasePipeline will just forward this on to each individual step so this same process happens in `sklearn`.

As a side not, the whole `if` statement wasn't necessary, `check_random_state` accounts for `None`.